src/wx/xrc/GeneralConfig.xrc

```diff
@@ -111,31 +111,31 @@
               <object class="sizeritem">
                 <object class="wxChoice" name="ThrottleSel">
                   <content>
-                    <item>Full Speed/No Limit</item>
-                    <item>25%</item>
-                    <item>50%</item>
-                    <item>75%</item>
-                    <item>100%</item>
-                    <item>125%</item>
-                    <item>150%</item>
-                    <item>175%</item>
-                    <item>200%</item>
-                    <item>225%</item>
-                    <item>250%</item>
-                    <item>275%</item>
-                    <item>300%</item>
-                    <item>325%</item>
-                    <item>350%</item>
-                    <item>375%</item>
-                    <item>400%</item>
-                    <item>425%</item>
-                    <item>450%</item>
-                    <item>475%</item>
-                    <item>500%</item>
-                    <item>525%</item>
-                    <item>550%</item>
-                    <item>575%</item>
-                    <item>600%</item>
+                    <item>Unlimited</item>
+                    <item translate="0">25%</item>
+                    <item translate="0">50%</item>
+                    <item translate="0">75%</item>
+                    <item translate="0">100%</item>
+                    <item translate="0">125%</item>
+                    <item translate="0">150%</item>
+                    <item translate="0">175%</item>
+                    <item translate="0">200%</item>
+                    <item translate="0">225%</item>
+                    <item translate="0">250%</item>
+                    <item translate="0">275%</item>
+                    <item translate="0">300%</item>
+                    <item translate="0">325%</item>
+                    <item translate="0">350%</item>
+                    <item translate="0">375%</item>
+                    <item translate="0">400%</item>
+                    <item translate="0">425%</item>
+                    <item translate="0">450%</item>
+                    <item translate="0">475%</item>
+                    <item translate="0">500%</item>
+                    <item translate="0">525%</item>
+                    <item translate="0">550%</item>
+                    <item translate="0">575%</item>
+                    <item translate="0">600%</item>
```

Comment:
Instead of selecting values for throttle, could we just display a text box allowing the user to set any value? We could have a button to set to default value (`100%`). This is only an idea.

Reply:
Yeah I will replace it with a textbox with some sort of on-change handler to round up large values.